OpenGov Treasury Doc (#4748)
w3f/polkadot-wiki: docs/learn/learn-polkadot-opengov-treasury.md

fees, slashing, and [staking inefficiencies](learn-staking.md#inflation).

Treasury funds are held in a [system account](./learn-account-advanced.md#system-accounts) that
cannot be controlled by any external account; only the system internal logic can access it.

## Treasury Inflow and Outflows

Tokens that are deposited into the Treasury (i.e. the inflow) is determined by the following
mechanisms:

- **Transaction fees:** 80% of the transaction fees of every submitted extrinsic is diverted to the
  Treasury, while 20% is given to the block producers.
- **Staking inefficiencies:** the network knows an exogenously determined parameter called ideal
  staking rate. The APY for stakers (nominators & validators) decreases whenever the actual staking
  rate is not equal to the ideal staking rate. To keep inflation constant at 10%, the system does
  not creates less tokens, rather some share of the overall reward for stakers is diverted to the
  Treasury (more information
  [here](https://research.web3.foundation/Polkadot/overview/token-economics)).
- **Slashes:** whenever validators and nominators are slashed, a share of the slashed tokens are

>   diverted to Treasury. This is a rare and unpredictable event.

Reviewer: ```suggestion
  diverted to Treasury. They are typically rare and unpredictable events.
```

> - **Transfers:** everyone can send funds to the Treasury directly. This is a rare event and mostly

Reviewer: ```suggestion
- **Transfers:** everyone can send funds to the Treasury directly. This is a rare event and typically
```
